example_test.go

```diff
@@ -7,8 +7,9 @@ package dque_test
 
 import (
 	"fmt"
-	"github.com/joncrlsn/dque"
 	"log"
+
+	"github.com/joncrlsn/dque"
```

Comment:
That was my VSCode editor (and one of the go-fmt / linters). Objections?

Reply:
No :-)